CONTRIBUTING.md

```diff
@@ -23,13 +23,14 @@ You should first [create a new issue](https://github.com/stylelint/stylelint/iss
 
 To start coding, you'll need:
 
--   a minimum of [Node.js](https://nodejs.org/en/) v10, though we do recommend using the latest LTS release
--   the latest [npm](https://www.npmjs.com/)
+- a minimum of [Node.js](https://nodejs.org/en/) v10, though we do recommend using the latest LTS release
+- the latest [npm](https://www.npmjs.com/)
 
 Then:
 
-1.  [Fork and clone](https://guides.github.com/activities/forking/) the stylelint repository.
-2.  Install all the dependencies with `npm ci`.
+1. [Fork and clone](https://guides.github.com/activities/forking/) the stylelint repository.
+
```

Comment:
Looks like empty line is not needed here.

Reply:
Good catch. I've added [remark-lint-list-item-spacing](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-list-item-spacing), which will catch these for us and also enforce a loose style for nested lists which are easier to read.